_layouts/sidebar.md

```diff
@@ -4,11 +4,17 @@
   - [How to add resources](../how-to-add-resources.md)
 * Resources
   - [DevOps](./../pages/devops-resources.md)
-  - [Web Development](./../pages/web-development-resources.md)
+  - <details>
+     <summary><a href="./../pages/web-development-resources.md">Web Development</a>
+     </summary>
+    
     - [HTML resources](./../pages/HTML-resources.md)
     - [CSS resources](./../pages/CSS-resources.md)
     - [React resources](./../pages/react-resources.md)
     - [Backend Development resources](./../pages/backend-development-resource.md)
+    - [Backend Development resources](./../pages/backend-development-resource.md) 
```

Comment:
The backend development resources have been duplicated twice @naikmubashir, please look into this and fix this